node/silkworm/etl/file_provider.cpp

```diff
@@ -104,8 +104,4 @@ void FileProvider::reset() {
     }
 }
 
-std::string FileProvider::get_file_name(void) const { return file_name_; }
-
-size_t FileProvider::get_file_size(void) const { return file_size_; }
-
```

Comment:
Please don't remove these. They're propedeutic to integration-2 where I implemented the total size and logging of collector

Reply:
Sure, restored.